src/RecordManager/Finna/Record/Ead3.php

```diff
@@ -480,6 +480,20 @@ protected function getAuthorIds()
         return $result;
     }
 
+    /**
+     * Get author ids for enrichment purposes.
+     *
+     * @return array
+     */
+    public function getAuthorIdsForEnrichment(): array
```

Comment:
Olisi parempi, että `getAuthorIds` hoitaisi homman finaf:n osaltakin, eikä tarvittaisi erillistä metodia. Muuten voi tulla melkoinen määrä ylimääräisiä tunnisteita tarkistettavaksi, ja toisaalta ne eivät indeksoidu tunnisteiksi Solr:iin. Jos metadatasta puuttuu jotain tämän toteuttamiseksi oikein, niin pitäisi korjata ensin metadata.

Reply:
Joo, kahtelin et onko tota authorsid:tä ees käytetty missään niin näyttäis siltä, että ei : | muokkasin sen sitten aika rajusti.